samples/app-dev/Connectors/Miscellaneous/ODataS4hana/readme.md

```diff
@@ -0,0 +1,253 @@
+![Sample Response](../../../import-screenshots/ODataS4Hana/31.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/32.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/33.png)
+
+Provide Json Response Schema in Output Settings tab.
+![Sample Response](../../../import-screenshots/ODataS4Hana/34.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/35.png)
+
+
+#### 4. Using Delete operation of OData Modify activity 
+
+We are deleting data from SAP S4 Hana 'A_ProductDescription' API service. Following is the configuration needed for OData Modify Activity using Delete operation.
+
+Select SAP S4 Hana connection in Settings tab of OData Modify activity. Also select Delete operation for deleting data from SAP S4 Hana service. By default Request Type application/json is selected.
+![Sample Response](../../../import-screenshots/ODataS4Hana/36.png)
+
+Add Request Body Schema for which you want to delete data from SAP S4 Hana service. Also add Request Headers parameter for mapping X-CSRF-Token value from OData Query activity response.
+![Sample Response](../../../import-screenshots/ODataS4Hana/37.png)
+
+Provide OData URI value for which you want to delete data from SAP S4 Hana service. Here we are deleting updated ProductDescription property value using ODataURI: "A_ProductDescription(Product='A001',Language='EN')"
+![Sample Response](../../../import-screenshots/ODataS4Hana/38.png)
+
+Provide value for requestBody
+![Sample Response](../../../import-screenshots/ODataS4Hana/39.png)
+
+Mapp x-csrf-token value of ODataQuery activity to the input of the x-csrf-token header of ODataModifyDelete activity. Similarly mapp set-cookie value of ODataQuery activity to the input of the Cookie header of ODataModifyDelete activity. Set Content-Type value as "application/json".
+![Sample Response](../../../import-screenshots/ODataS4Hana/40.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/41.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/42.png)
+
+Provide Json Response Schema in Output Settings tab.
+![Sample Response](../../../import-screenshots/ODataS4Hana/43.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/44.png)
+
+Also in flow we have Log Message for every activity output and in last Return Activity for getting all activity output.
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/45.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/46.png)
+
+### Run the application
+For running the application, first you have to push the app and then scale up the app.
+Then after sometime you can see your app goes in running state.
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/47.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/48.png)
```

Comment:
remove this also and check if we remove some more which are not needed... too many screenshots make sample unreadable.